### PR TITLE
Fix expert template

### PR DESCRIPTION
**Pull request by deivid-rodriguez**
When trying to build the sample app, I found this errors.

### CODE REVIEW

**Comment by karmi** [comment]
Ah, sorry, fixed some of the issues in other commits... Wondering about the `require` fix, will add line note.

File: elasticsearch-rails/lib/rails/templates/seeds.rb [hunk 1]
```diff
@@ -53,4 +53,5 @@
 
 # Remove any jobs from the "elasticsearch" Sidekiq queue
 #
+require 'sidekiq/api'
```
Comment (karmi) [inline elasticsearch-rails/lib/rails/templates/seeds.rb]: Weird -- I've just run the template and all goes fine, what error did you get?

Comment (deivid-rodriguez) [inline elasticsearch-rails/lib/rails/templates/seeds.rb]: I got

    uninitialized constant Sidekiq::Queue (NameError)

Where else are you requiring `sidekiq/api`? To me the error and the fix seemed perfectly normal.

**Comment by karmi** [comment]
That's why I'm asking about it -- even running bare `rake db:seeds` in the generated application folder works (note lacking `bundle exec`). I think internally the `db:seeds` task invokes the `environment` task, and so Sidekiq is loaded as part of that...?

**Comment by karmi** [comment]
Can you provide more details about how you run it, what is the Ruby and Rails version you have?

**Comment by deivid-rodriguez** [comment]
I'm going to install the sample app again and see if it still happens.

My Ruby and Rails versions are both the latest releases.

Notice that `sidekiq/api` is not required by default by `sidekiq`. At the least that's what I gather from [the docs](https://github.com/mperham/sidekiq/wiki/API) and what I experience creating a sample app, including `sidekiq` in the Gemfile and running stuff like

    bundle exec ruby -r sidekiq -e 'Sidekiq::^Ceue.new("elasticsearch")'
    # => Fails with a name error

    bundle exec ruby -r "sidekiq/api" -e 'Sidekiq::Queue.new("elasticsearch")'
    # => Runs fine

**Comment by karmi** [comment]
Thanks! I mean the `require` statement is pretty harmless, but I'd like to understand more what's going on...

**Comment by deivid-rodriguez** [comment]
@karmi Yep, same error

    rake aborted!
    NameError: uninitialized constant Sidekiq::Queue
    /home/davidr/Work/searchapp/db/seeds.rb:56:in `<top (required)>'
    /home/davidr/.gem/ruby/2.2.1/gems/railties-4.2.1/lib/rails/engine.rb:547:in `load_seed'
    /home/davidr/.gem/ruby/2.2.1/gems/activerecord-4.2.1/lib/active_record/tasks/database_tasks.rb:250:in `load_seed'
    /home/davidr/.gem/ruby/2.2.1/gems/activerecord-4.2.1/lib/active_record/railties/databases.rake:180:in `block (2 levels) in <top (required)>'
    /home/davidr/.gem/ruby/2.2.1/gems/activerecord-4.2.1/lib/active_record/railties/databases.rake:139:in `block (2 levels) in <top (required)>'
    Tasks: TOP => db:setup => db:seed
    (See full trace by running task with --trace)

I did

    rm -rf searchapp
    rails new searchapp --skip --skip-bundle --template https://raw.github.com/elasticsearch/elasticsearch-rails/master/elasticsearch-rails/lib/rails/templates/01-basic.rb
    rails new searchapp --skip --skip-bundle --template https://raw.github.com/elasticsearch/elasticsearch-rails/master/elasticsearch-rails/lib/rails/templates/02-pretty.rb
    rails new searchapp --skip --skip-bundle --template https://raw.github.com/elasticsearch/elasticsearch-rails/master/elasticsearch-rails/lib/rails/templates/03-expert.rb

**Comment by karmi** [comment]
@deivid-rodriguez Merged adding the require, thanks! In the near future, I really need to add tests for the generated apps, and run them in CI.

BTW, for an "unsupervised" installation, you can use the environment variables to not install ES and not start the app: `RAILS_NO_ES_INSTALL=y RAILS_NO_SERVER_START=y`

**Comment by deivid-rodriguez** [comment]
Thanks @karmi !

**Pull request closed without merging**